readme.md

```diff
@@ -0,0 +1,30 @@
+# OlympusExtra
+
+OlympusExtra is a Hades DLC mod which adds new gods and boons to the Hades game. This includes unique effects, new god curses, duo boons, Thesus's wrath calls, and more. 
+
+## Locating your Hades game folder for modding
+- **Steam**: `Library` > Right click on Hades > `Manage` > `Browse local files` 
+- **Epic Games**: Go to `C:\Program Files\Epic Games\Hades`
+
+-----------------------------
+## Pre-Requisites 
+1. Download and unzip [modimporter](https://github.com/SGG-Modding/sgg-mod-modimporter/releases/latest/download/modimporter-windows.zip) (follow installation instructions on the Nexus Mod page). The downloaded `modimporter.exe` should be placed in the `Hades\Content` folder
+1. Download and unzip [Mod Utility](https://github.com/SGG-Modding/sgg-mod-modutil/releases/download/v2.3.1/ModUtil.zip). Place the unzipped contents in the `Hades\Content\Mods` folder 
+
+-----------------------------
+## Installation Instructions
+1. Make sure you fulfill the above pre-requisites!
+1. Download the OlympusExtra mod from the [Releases](https://github.com/AlexKage69/OlympusExtra/releases) section 
+1. Place the unzipped contents in the `Hades\Content\Mods` folder
+1. Run the `Hades\Content\modimporter.exe` executable 
+1. Open Hades and start playing!
+
+-----------------------------
+## ApolloExtra 
+Introducing Lord Apollo: God of Music, the Sun, and Healing! 
+
+Apollo's unique curse is Blind: foes inflicted with blind have a percentage chance to miss their next attack. 
+
+Do note that we're still in beta-testing phase for ApolloExtra, not all features have been fully implemented yet! We welcome your feedback on the [Hades Modding Discord](https://discordapp.com/invite/KuMbyrN)
+
```

Comment:
We could list the intended DLC content. Like: Coming next: Hestia, Hera, Hephaestus, Gaia and more.